ppa/archive/models.py

```diff
@@ -401,18 +460,34 @@ def hathi_content_dir(self):
         # pairtree encoded version of the id
         return pairtree_path.id_encode(self.hathi_pairtree_id)
 
+    def hathi_pairtree_client(self):
+        '''Initialize a pairtree client for the pairtree datastore this
+         object belongs to, based on its Hathi prefix id.'''
+        return pairtree_client.PairtreeStorageClient(
+            self.hathi_prefix,
+            os.path.join(settings.HATHI_DATA, self.hathi_prefix))
+
     def hathi_pairtree_object(self, ptree_client=None):
```

Comment:
That's still used by the `hathi_import` script - I added it so I can initialize one pairtree client for each pairtree datastore for the duration of the script, rather than having to repeatedly initialize clients for each object.

Reply:
Makes sense. I assumed the answer would be yes, but it was the only spare thing I saw in the code so mentioned it.

Reply:
Worth documenting anywhere? I'm not sure where would make sense, but might be good to have somewhere in this file.

Reply:
Maybe in a Sphinx docstring with the param?